recipes/libfdk_aac/all/conanfile.py

```diff
@@ -46,6 +48,12 @@ def configure(self):
         if self.options.shared:
             self.options.rm_safe("fPIC")
 
+    def validate(self):
```

Comment:
```suggestion
    def validate_build(self):
```

Following @AbrilRBS 's suggestion. Indeed anyone could consume a package produced by Android. The restriction is when cross-building, using Android NDK only.